frontend/components/LiveQuery/SelectTargets.tsx

```diff
@@ -253,6 +279,50 @@ const SelectTargets = ({
     }
   );
 
+  // Ensure that the team or label list is expanded on the first load only if a hidden entity is already selected
+  const shouldExpandList = (
+    list: ISelectLabel[] | ISelectTeam[],
+    truncatedList: ISelectLabel[] | ISelectTeam[]
+  ) => {
+    return list.some(
+      (entity) =>
+        !truncatedList.some(
+          (truncatedEntity) => truncatedEntity.id === entity.id
+        )
+    );
+  };
+
+  const expandListsOnInitialLoad = () => {
```

Comment:
The "initial" this refers to is the rendering of this component? There won't ever be any targets
selected on initial load of the page right?

Reply:
In code clarification comment is on function directly above this one https://github.com/fleetdm/fleet/pull/24798#discussion_r1890950151